src/sentry/static/sentry/app/views/organizationStats/redesign/index.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router';
+import styled from '@emotion/styled';
+import capitalize from 'lodash/capitalize';
+
+import {Client} from 'app/api';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
+import Card from 'app/components/card';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import PageHeading from 'app/components/pageHeading';
+import {Panel, PanelBody} from 'app/components/panels';
+import {t, tct} from 'app/locale';
+import {PageContent, PageHeader} from 'app/styles/organization';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+
+import {OrganizationUsageStats, ProjectUsageStats} from './types';
+
+type Props = {
+  api: Client;
+  organization: Organization;
+} & RouteComponentProps<{orgId: string}, {}>;
+
+type State = {
+  dataCategory: DataCategory;
+
+  orgStats?: OrganizationUsageStats;
+  orgStatsLoading: boolean;
+  orgStatsError?: Error;
+
+  projectStats?: ProjectUsageStats[];
+  projectStatsLoading: boolean;
+  projectStatsError?: Error;
+};
+
+// TODO: Move over from getsentry
+export enum DataCategory {
+  ERRORS = 'errors',
+  TRANSACTIONS = 'transactions',
+  ATTACHMENTS = 'attachments',
+}
+
+class OrganizationStats extends React.Component<Props, State> {
+  state: State = {
+    dataCategory: DataCategory.ERRORS,
+    orgStatsLoading: false,
+    projectStatsLoading: false,
+  };
+
+  componentDidMount() {
+    this.getOrganizationStats();
+    this.getProjectsStats();
+  }
+
+  setSelectedDataCategory = (dataCategory: DataCategory) => {
+    this.setState({dataCategory});
+  };
+  get selectedDataCategory() {
+    return capitalize(this.state.dataCategory);
+  }
+
+  /**
+   * Fetches aggregated stats of tne entire organization
+   */
+  getOrganizationStats() {
+    this.setState({orgStatsLoading: true});
+
+    const orgStats: OrganizationUsageStats = {
+      statsErrors: [],
+      statsTransactions: [],
+      statsAttachments: [],
+    };
+
+    for (let i = 0; i < 31; i++) {
+      const stats = {
+        ts: i.toString(),
+        accepted: {timesSeen: i * 100, quantity: i * 1000},
+        filtered: {timesSeen: i * 100, quantity: i * 1000},
+        dropped: {
+          overQuota: {timesSeen: i * 100, quantity: i * 1000},
+          spikeProtection: {timesSeen: i * 100, quantity: i * 1000},
+          other: {timesSeen: i * 100, quantity: i * 1000},
+        },
+      };
+
+      orgStats.statsErrors.push(stats);
+      orgStats.statsTransactions.push(stats);
+      orgStats.statsAttachments.push(stats);
+    }
+
+    setTimeout(() => {
+      this.setState({
+        orgStatsLoading: false,
+        orgStats,
+      });
+    }, 3000);
+  }
+
+  /**
+   * Fetches stats of projects that the user has access to
+   */
+  getProjectsStats() {
+    return [];
+  }
```

Comment:
Placeholder until [ER-605](https://getsentry.atlassian.net/browse/ER-605) is completed.